caffe2/python/tutorials/py_gen/MNIST.py

```diff
@@ -128,20 +152,52 @@ def AddInput(model, batch_size, db, db_type):
     return data, label
 
 
-# At this point we need to take a look at the predictions coming out of the network at convert them into probabilities.
-# "What's the probability that this number we're looking at is a 5", or "is this a 7", and so forth. 
+# Now we are going to construct our own model. In this part input is our data blob. The output is a blob which will contain probabilities for each of the 10 possible digits.
```

Comment:
The input will be our data blob, and the output will be vectors of length 10 containing the network's prediction on each of the 10 possible digits.